integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -534,3 +556,28 @@ In order to ingest to Timestream for LiveAnalytics, every line protocol point mu
 
 The connector expects query string parameters to be included as `queryParameters` or `queryStringParameters` in requests.
 
+### Stack Cost
+
+The following is an overview of how the costs are calculated for each deployed resource in the CloudFormation stack, at the time of writing (Oct. 21, 2024). These calculations are based on the calculations used by the [AWS pricing calculator](https://calculator.aws/#/). Refer to the AWS pricing calculator for a more accurate estimate:
+
+- Lambda (without free tier):
+    - number of monthly requests x average request duration x 0.001 ms to sec conversion factor = total compute in seconds.
+    - memory usage in GB x total compute in seconds = total compute GB-s.
+    - total compute GB-s x 0.0000133334 USD = tiered price.
+    - number of monthly requests x 0.0000002 USD = monthly request charges.
+    - tiered price + monthly request charges = Lambda monthly cost.
+- REST API Gateway:
+    - number of monthly requests x 0.0000035 USD = REST API Gateway monthly cost.
+- CloudWatch:
+    - logs data ingested in GB per month x 0.50 USD = logs data ingested cost per month.
+    - logs data ingested in GB per month x 0.15 Storage compression factor x 1 Logs retention factor x 0.03 USD = standard/vended logs data storage cost.
+    - logs data ingested cost + standard/vended logs data storage cost = CloudWatch monthly cost.
+
+#### Solution
+
+The following are some approaches that can reduce stack costs:
+
+- The REST API Gateway incurs the highest costs for the stack. Reduce REST API Gateway costs by including as many line protocol points in a request as possible. 500-20,000 line protocol points in each request is ideal. The Lambda memory size, Lambda timeout, REST API Gateway timeout, and client timeout may have to be increased in order to accommodate larger requests.
+    - NOTE: if requests contain only a single line protocol point, the ratio of REST API Gateway requests to ingested records would be 1:1 and costs would be much higher than including multiple line protocol points in each request.
+- If possible, create the necessary tables in advance, to reduce the time the connector will take to create tables. The connector adds a one second delay for table or database creation in order to avoid throttling.
```

Comment:
Should we add a note to disable DB/Table creation to avoid API calls to check if they exist?

Reply:
Added:
```markdown
- Set the `EnableDatabaseCreation` or `EnableTableCreation` parameter to `false` to skip checks for existing databases or tables, if unnecessary.
```